content/doc/book/managing/system-properties.adoc

```diff
@@ -1830,6 +1830,17 @@ properties:
     Setting this to `true` is unsafe.
     See link:/redirect/class-filter/[documentation].
 
+- name: jenkins.security.FIPS140.COMPLIANCE
+  tags:
+  - feature
+  - security
+  def: |
+    `false`
+  since: 2.424
+  description: |
+    Whether Jenkins and plugins system should prefer link:https://csrc.nist.gov/pubs/fips/140-2/upd2/final[FIPS-140] compliant cryptography.
```

Comment:
```suggestion
    The use of link:https://csrc.nist.gov/pubs/fips/140-2/upd2/final[FIPS-140] compliant cryptography for Jenkins and plugins systems.
```

Reply:
the suggestion does not correlate with what this flag does.  It does not cause any change, it is merely there for code to look at to decide if it wants to change (not all code will look at this flag) - so implying that this uses compliant cryptography would be incorrect and something that we should steer away from.

Reply:
Revised my comment.